### DISCUSSION TITLE
Rest API?

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#1860] [🇩🇪 Fragen&Antworten] [answered]
Guten Tag Zusammen

Ich wollte kurz fragen, da ich nirgends eine definitve Antwort fand, existiert eine Rest API mit der man die Daten lesen, sowie auch die Geräte steuern kann?

Danke fürs Antworten.

Freundliche Grüsse
TD3V

**Comment by jens-maus** [comment, marked as answer]
Für diesen Zweck gibt es für RaspberryMatic bzw. eine CCU das Third-Party Add-on "CCU-Jack". Das sollte eine entsprechende RestAPI mit sich bringen mit der man entsprechende Geräteabfragen dann tätigen kann:

https://github.com/mdzio/ccu-jack
